icaruscode/TPC/Tracking/ICARUSPandora/CMakeLists.txt

```
@@ -1,7 +1,44 @@
 
 # where should the scripts/..xml file be installed?  Perhaps in bin?
 
+set( TOOL_LIBRARIES
+	larpandora::LArPandora
+	larpandora::GeometryComponents
+	larpandoracontent::LArPandoraContent
+	lardata::AssociationUtil
+	lardata::Utilities
+	lardataobj::AnalysisBase
+	lardataobj::RecoBase
+	lardataobj::Simulation
+	larcoreobj::SimpleTypesAndConstants
+	nusimdata::SimulationBase
+	art::Persistency_Common
+	canvas::canvas
+	PandoraPFA::PandoraSDK
+	larpandora::GetDetectorType
+	larreco::ClusterFinder
+	larreco::ClusterParamsImportWrapper
+	larreco::RecoAlg_ClusterRecoUtil
+	larevt::ChannelStatusProvider
+	larevt::ChannelStatusService
+	larsim::MCCheater_ParticleInventoryService_service
+	lardata::DetectorClocksService
+	lardata::DetectorPropertiesService
+	larcore::Geometry_Geometry_service
+	larcorealg::Geometry
+	art_root_io::TFileService_service
+	art::Framework_Principal
+	art::Framework_Services_Registry
+	messagefacility::MF_MessageLogger
+	cetlib::cetlib
+	cetlib_except::cetlib_except
+	PandoraPFA::PandoraSDK
```

I think these are the ones needed for this tool:
```suggestion
	larpandora::LArPandoraInterface
	lardataobj::RecoBase
	canvas::canvas
	art::Framework_Principal
	messagefacility::MF_MessageLogger
	cetlib::cetlib
```

---

Cool, I'll try to build with this reduced set! 🤞

---

Addressed. Thanks for the suggestion!